app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+            self, distance_from_city_center: (int or float), clean_power: int,
```

Comment:
distance_from_city_center: (int or float) is a bad typing
use:
distance_from_city_center: int | float      for python 3.10+
or
distance_from_city_center: Union[int, float]